controllers/activegate/capability/capability.go

```diff
@@ -57,12 +73,12 @@ func (c *capabilityBase) GetConfiguration() Configuration {
 	return c.Configuration
 }
 
-func (c *capabilityBase) GetModuleName() string {
-	return c.moduleName
+func (c *capabilityBase) GetShortName() string {
```

Comment:
```suggestion
func (c *capabilityBase) ShortName() string {
```

Reply:
then I should redo all the other functions in the interface, which will result in more files changed, if you are up for it then I will do it 😛